<launch>

  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="TestNeck" />

</launch>
